README.md

```diff
@@ -1,23 +1,45 @@
-# PactAnnotationBuilder
+# Pact Builder DSL
 
 [![Codacy Badge](https://app.codacy.com/project/badge/Grade/67f7406f9b79477faae81cc93ed79395)](https://app.codacy.com/gh/sngular/pact-annotation-processor/dashboard?utm_source=gh&utm_medium=referral&utm_content=&utm_campaign=Badge_grade)[![Maven Central](https://img.shields.io/maven-central/v/com.sngular/pact-annotation-processor?label=Maven%20Central)](https://search.maven.org/search?q=g:%22com.sngular%22%20AND%20a:%22pact-annotation-processor%22)
 
-This is an annotation processor designed to generate Pact Dsl Builders based on annotated classes.
+This is a helper tool for Contract Testing with [Pact](https://docs.pact.io/).
+An annotation processor designed to generate DslPart objects for the body definitions based on annotations included in your model classes.
 
-## Index
+The focus is to simplify JVM contract testing implementation, minimizing the amount of boilerplate code needed.
 
-- [Pact Annotation Builder](#pact-annotation-builder)
-- [Main Configuration](#main-configuration)
-  - [How to add the annotations](#how-to-add-the-annotations)
-  - [How to use the Builders](#how-to-use-the-builders)
+Specially useful when defining body validations for interactions with complex models.
 
-## Main Configuration
+## Table of Contents
 
-This plugin allows developers to automatize the creation of code for build Pact Request Bodies for Pact Consumer Contracts.
+- [Pact Builder DSL](#pact-builder-dsl)
+- [Getting Started](#getting-started)
+  - [Requirements](#requirements)
+  - [Compatibility with pact-jvm](#compatibility-with-pact-jvm)
+  - [Configuration](#configuration)
+- [Usage](#usage)
+  - [Annotations](#annotations)
+  - [Builder](#builder)
+- [Roadmap](#roadmap)
+- [Contributing](#contributing)
+- [License](#license)
+- [Contact](#contact)
 
-### How to configure the POM file
+## Getting Started
+### Requirements
+- JDK +17
+- Having in project a verification library from your choice to have @Max and @Min annotations available (like [Jakarta](https://central.sonatype.com/artifact/jakarta.validation/jakarta.validation-api), [Spring Boot](https://central.sonatype.com/artifact/org.springframework.boot/spring-boot-starter-validation), or similar)
```

Comment:
Having inside your project a verification library of your choice so as to have the `@Max` and `@Min` annotations available, such as Jakarta, Spring Boot or similar.

Reply:
Applied, thanks!